packages/adyen-utils/src/types.ts

```diff
@@ -943,7 +1028,21 @@ export function isBoletoState(param: unknown): param is BoletoState {
     );
 }
 
-export interface AdyenPaymentMethodInitializationData {
+export function isAccountState(param: any): param is AccountState {
```

Comment:
I have 2 questions around this typeguard:
1. Should we keep it in types file or better move it to a new file?
2. Is it possible to change `any` type to something else?